polaris-core/src/main/java/org/apache/polaris/core/config/FeatureConfiguration.java

```diff
@@ -91,6 +91,19 @@ public static void enforceFeatureEnabledOrThrow(
           .defaultValue(false)
           .buildFeatureConfiguration();
 
+  public static final FeatureConfiguration<Boolean> INCLUDE_SESSION_TAGS_IN_SUBSCOPED_CREDENTIAL =
+      PolarisConfiguration.<Boolean>builder()
+          .key("INCLUDE_SESSION_TAGS_IN_SUBSCOPED_CREDENTIAL")
+          .description(
+              "If set to true, session tags (catalog, namespace, table, principal, roles) will be included\n"
+                  + "in AWS STS AssumeRole requests for credential vending. These tags appear in CloudTrail events,\n"
+                  + "enabling correlation between catalog operations and S3 data access.\n"
+                  + "Requires the IAM role trust policy to allow sts:TagSession action.\n"
+                  + "Note that enabling this feature leads to degradation in temporary credential caching as \n"
+                  + "catalog will no longer be able to reuse credentials for different tables/namespaces/roles.")
```

Comment:
```suggestion
                  + "Note that enabling this feature may lead to degradation in temporary credential caching as \n"
                  + "catalog will no longer be able to reuse credentials for different tables/namespaces/roles.")
```

i see allowedLocation is part of the table for general case we have overlapping check, having MAY is neutral in a sense it will not scare folks. 


though i am not sure why just principal is in the key and not the roles, @dimas-b you know why ?